src/styles/header.scss

```diff
@@ -0,0 +1,66 @@
+.header {
+  max-width: 1200px;
+  display: flex;
+  font-family: Roboto, system-ui;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 12px;
+  font-weight: 500;
+  position: relative;
+  padding: 0 50px;
+  margin: 0 auto;
+
+  &__logo {
+    display: block;
+    height: 40px;
+    width: 40px;
+  }
```

Comment:
The logo width and height are both set to 40px. If the logo is not a square, this could lead to distortion. It's usually better to set only one dimension (width or height) and let the other adjust automatically to maintain the aspect ratio.